ccloader/js/mod.js

```diff
@@ -149,6 +183,79 @@ export class Mod {
 		this.runtimeAssets[original] = newPath;
 	}
 
+	/**
+	 * Returns the full path to a resource
+	 * @param {string} relativePath to resource with mod folder as base
+	 * @returns {string} 
+	 */
+	absolutePath(relativePath) {
```

Comment:
This function "`absolutePath`" should either return a path starting with `/` relative to the game root folder (where it's exe is) for consumption in nodejs or an url that includes `location.origin` to make it absolute. If this function is only intended for use in `_findAssets` then it should be made private.

Reply:
This is not just for internal use. The entire usage is for mods that gets resources from other mods.